functions/concepts/filesystem.rb

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START functions_concepts_filesystem]
+require "functions_framework"
+require "cgi"
+
+FunctionsFramework.http "concepts-filesystem" do |_request|
+  files = Dir.entries "."
+  files_escaped = files.map { |f| CGI.escape_html f }
```

Comment:
Why do we need to escape the names? The python and node samples don't seem to do so.